bigtable-dataflow-import/src/test/java/com/google/cloud/bigtable/dataflowimport/testing/BigtableTableUtils.java

```diff
@@ -109,10 +128,17 @@ public void close() throws IOException {
     }
 
     public static BigtableTableUtilsFactory from(HBaseImportOptions options) throws IOException {
-      return new BigtableTableUtilsFactory(BigtableConfiguration.connect(
-          options.getBigtableProjectId(),
-          options.getBigtableZoneId(),
-          options.getBigtableClusterId()));
+      BigtableConnection connection = new BigtableConnection(
+          new CloudBigtableConfiguration.Builder()
+              .withProjectId(options.getBigtableProjectId())
+              .withClusterId(options.getBigtableClusterId())
+              .withZoneId(options.getBigtableZoneId())
+              .withConfiguration(
+                  "hbase.client.connection.impl",
+                  "com.google.cloud.bigtable.hbase1_0.BigtableConnection")
```

Comment:
I may have a git merge accident. Thanks for catching it. Will revert this part to master's.